feedback/lib/src/feedback_bottom_sheet.dart

```diff
@@ -103,8 +103,7 @@ class _DraggableFeedbackSheetState extends State<_DraggableFeedbackSheet> {
         ),
         Expanded(
           child: DraggableScrollableSheet(
-            // Should be enabled as soons as the Flutter version gets raised
-            // snap: true,
+            snap: true,
```

Comment:
Please raise the minimal required Flutter version in the pubspec. Otherwise this breaks users on older Flutter versions without a good error message.

Reply:
Fix'd.